INSTALL.rst

```diff
@@ -1,27 +1,59 @@
 Installation
 ============
 
+Zenodo depends on PostgreSQL, Elasticsearch 2.x, Redis and RabbitMQ.
+
+If you are only interested in running Zenodo locally, follow the Docker
+installation guide below. If you plan to eventually develop Zenodo code you
+continue further to Development installation to find out how to set up the
+local instance for easy code development.
+
+For this guide you will need to install
+`docker <https://docs.docker.com/engine/installation/>`_ along with the
+`docker-compose <https://docs.docker.com/compose/>`_ tool.
+
+Docker installation is not necessary, although highly recommended.
+
+If you can't use docker you can run Zenodo and all of the required services
+directly in your system. Take a look at ``docker-compose.yml`` file to find out
+what is required and how the configuration looks like.
+For development you will need to set-up an configure
+four services: PostgreSQL (``db``), Elasticsearch (``es``),
+Redis (``cache``) and RabbitMQ (``mq``).
+
 Docker installation
 -------------------
-The easiest way to get started with Zenodo is using the provided docker image.
-First checkout the source code, build all docker images and boot them up
-using Docker Compose:
+The easiest way to run Zenodo locally is to use the provided docker-compose
+configuration containing full Zenodo stack. First checkout the source code,
+build all docker images and boot them up using ``docker-compose``:
 
 .. code-block:: console
 
+    $ cd ~/src/
     $ git clone https://github.com/zenodo/zenodo.git
+    $ cd zenodo
     $ git checkout master
     $ docker-compose build
     $ docker-compose up
 
-Next, create the database, indexes, fixtures and an admin user:
+
+.. note::
+
+    For the purpose of this guide we will assume that all repositories are
+    checked into ``~/src/`` directory.
+
+
+Keep the session with the docker-compose above alive, and in a separate shell
+run the init script which creates the database tables, search indexes
+and some data fixtures:
 
 .. code-block:: console
 
+    $ cd zenodo
```

Comment:
Since this is already done in the previous code block it may confuse people who will end up inside the `zenodo/zenodo` directory.